### PR TITLE
Add whitelist for syncable owners

### PR DESCRIPTION
**Pull request by imduffy15**

### CODE REVIEW

File: router/middleware/config.go [hunk 1]
```diff
@@ -39,6 +39,7 @@ func setupConfig(c *cli.Context) *model.Settings {
 		Secret: c.String("agent-secret"),
 		Admins: sliceToMap2(c.StringSlice("admin")),
 		Orgs:   sliceToMap2(c.StringSlice("orgs")),
+		OwnersWhitelist: sliceToMap2(c.StringSlice("repo-owner")),
```
Comment (laszlocph) [inline router/middleware/config.go:42]: Does this work? `repo-owners` is specified at https://github.com/laszlocph/woodpecker/pull/119/files#diff-a1e14c06700b04b423b787fd33d25b34R107 
Mind the plural form

File: server/user.go [hunk 2]
```diff
@@ -45,10 +45,13 @@ func GetFeed(c *gin.Context) {
 		user.Synced = time.Now().Unix()
 		store.FromContext(c).UpdateUser(user)
 
+		config := ToConfig(c)
+
 		sync := syncer{
 			remote: remote.FromContext(c),
 			store:  store.FromContext(c),
 			perms:  store.FromContext(c),
+			match: NamespaceFilter(config.Orgs),
```
Comment (laszlocph) [inline server/user.go:54]: Looks like `OwnersWhitelist` is never used, nor the related configuration variable. Looks like you abandoned that concept and just used `config.Orgs`? If I think about it, that var is probably enough

**Comment by imduffy15** [comment]
Thanks for the review @laszlocph, I mixed up some different commits/iterations when splitting things up into PRs, didn't spot the errors on this one. Thanks for highlighting them will adjust.

File: server/user.go (hunk 2 above)
Comment (imduffy15) [reply server/user.go]: Adjusted! Sorry about that, missed a few things when splitting the features up into separate PRs. Will go over the other PRs too just to be sure I didn't miss anything else.

Comment (laszlocph) [reply server/user.go]: No worries, this is stellar stuff!

Comment (imduffy15) [reply server/user.go]: I feel the same about your fork! It was fantastic to find it. I was previously running drone 0.2 and in need of an upgrade went to the latest version and only learned about the unusual license (given all the code is available) when looking at the autoscaling component.

Found your project via your blog post detailing a similar license confusion. Happy to contribute to such a lightweight, flexible CI solution that doesn't do pricing based on seats and/or repos/builds.

**Pull request merged**